osu.Game/Graphics/TimeDisplay.cs

```diff
@@ -0,0 +1,39 @@
+﻿//Copyright (c) 2007-2016 ppy Pty Ltd <contact@ppy.sh>.
+//Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using System;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Sprites;
+
+namespace osu.Game.Graphics.TimeDisplay
+{
+    public class TimeDisplay : Drawable
+    {
+        private SpriteText timeText;
+        private bool _24Hour;

```

Comment:
Which one?

timeText is needed to set the display text and _24Hour is needed to keep track of whether to display the 24 hour format or not.